### PR TITLE
Team2 Contributions

### PR DESCRIPTION
**Pull request by jnormile**
## Description

Team 2's cumulative proposed changes

### Type of Change

- [ ] Feature
- [ ] Bug fix
- [x] Documentation

## Contributors

- @jnormile 
- @AidanNeeson 
- @ningerson2002 
- @gormley01 
- @brendengray 

## Reminder

All GitHub Actions checks must be in a passing state in this branch or fork before this pull request can be merged.

### CODE REVIEW

**Review by favourojo — APPROVED** [review]
All looks good to me!

File: conduct.md [hunk 1]
```diff
@@ -36,31 +36,39 @@ TODO: Write an introductory paragraph.
...
+Communication is the cornerstone of collaboration, and all communications for team-based activities in the Software Engineering course must adhere to the below guidelines for professionalism. To be clear, we recognize two platforms for our official communication: Discord and GitHub. That having been said, the below guidelines ought to be followed for any in-person communication pertaining to or within the classroom.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are encouraged to:
+
+- Be proactive about communicating. This can include utilizing whatever platforms necessary to let other know ahead of time of any kinds of changes, concerns, or ideas.
```
Comment (burgess01) [inline conduct.md]: change 'other' to 'others'

**Review by solisa986 — APPROVED** [review]
Looks good! Thank you @jnormile @AidanNeeson @ningerson2002 @brendengray @gormley01 for your hard work on this!

File: conduct.md [hunk 2]
```diff
@@ -36,31 +36,39 @@ TODO: Write an introductory paragraph.
...
+- Constructively criticize others. For example, students should give honest and beneficial suggestions that do not alienate others and their work, which will encourage later participation.
+
+- Acknowledge others. Students should not isolate themselves or isolate others from any form of communication. Students should pull one another, and themselves, into the loop.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+Students are discouraged to:
```
Comment (burgess01) [inline conduct.md]: Maybe 'discouraged from' would flow better? That is personal preference though, if you want to keep it as is thats fine!

File: conduct.md [hunk 3]
```diff
@@ -75,13 +83,13 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
+- Create decision that mutually benefit each other (Schedules, Coding). This is to help understand each other's responsibilities and respect on what is possible for both individual interest and the team interest.
```
Comment (burgess01) [inline conduct.md:89]: Change 'decision' to 'decisions'. Also, please reword the second sentence.

**Review by ColeKaupelis — APPROVED** [review]

File: conduct.md [hunk 4]
```diff
@@ -75,13 +83,13 @@ TODO: Write an introductory paragraph.
...
 - TODO
 - TODO
 
 #### Unacceptable Conduct
 
-- TODO
+- Never informing on others on unattended appointments. This will only create more problems as other team members has to do more work unexpectedly, therefore the work will not be as good as intended to be.
```
Comment (burgess01) [inline conduct.md:96]: Please reword this, especially the first sentence.

File: conduct.md [hunk 5]
```diff
@@ -91,13 +99,13 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
+- Always be transparent on what are we doing wrong with. The reason being is because to create a mutual relationship environment between the team to understand the good and the bad of the activity together.
```
Comment (burgess01) [inline conduct.md]: Please reword this.

File: conduct.md [hunk 6]
```diff
@@ -91,13 +99,13 @@ TODO: Write an introductory paragraph.
...
 - TODO
 - TODO
 
 #### Unacceptable Conduct
 
-- TODO
+- Never create unnecessary mistakes (Forget to git pull, never creating pull requests). This is very important to not create more problems and not to halt on the team's progress on the work.
```
Comment (burgess01) [inline conduct.md:114]: Maybe change 'This is very important to not create more problems' to something like 'It is very important not to create more problems'.

**Review by burgess01 — CHANGES_REQUESTED** [review]
This looks great! Please fix these errors and I think it should be ready to be merged.

File: conduct.md [hunk 7]
```diff
@@ -39,31 +39,39 @@ The deadline for each task should be 12am. This will help other contributors to
 
 ### Communication
 
-TODO: Write an introductory paragraph.
+Communication is the cornerstone of collaboration, and all communications for team-based activities in the Software Engineering course must adhere to the below guidelines for professionalism. To be clear, we recognize two platforms for our official communication: Discord and GitHub. That having been said, the below guidelines ought to be followed for any in-person communication pertaining to or within the classroom.
```
Comment (Darius90332) [inline conduct.md:42]: Maybe emphasize how important communication will be considering teams won't always be the same

**Review by Darius90332 — APPROVED** [review]

**Comment by benfarrah01** [comment]
Everything looks good to me! Nice job, these are good points for the code of conduct.

**Pull request merged**